utils/profile_tool/sub.py

```diff
@@ -0,0 +1,43 @@
+import os
+import jinja2
+from ssg.build_yaml import Profile
+from ssg.environment import open_environment
+
+
+def command_sub(args):
+    product_yaml = os.path.join(args.ssg_root, "products", args.product, "product.yml")
+    env_yaml = open_environment(args.build_config_yaml, product_yaml)
+    try:
+        profile1 = Profile.from_yaml(args.profile1, env_yaml)
+        profile2 = Profile.from_yaml(args.profile2, env_yaml)
+    except jinja2.exceptions.TemplateNotFound as e:
+        print("Error: Profile {} could not be found.".format(str(e)))
+        exit(1)
+
+    subtracted_profile = profile1 - profile2
+
+    exclusive_rules = len(subtracted_profile.get_rule_selectors())
+    exclusive_vars = len(subtracted_profile.get_variable_selectors())
+    if exclusive_rules > 0:
+        print("{} rules were left after subtraction.".format(exclusive_rules))
```

Comment:
I don't see a problem to print these lines even if the number is 0. Therefore, we could move these two prints to `if exclusive_rules > 0 or exclusive_vars > 0:` and remove some unnecessary conditionals.